package.json

```diff
@@ -4,17 +4,17 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "yarn start:dev",
-    "start:gen": "node server.js",
-    "start:dev": "json-server --watch dev_db.json --routes routes.json --middlewares ./middleware.js",
-    "start:prod": "json-server db.json --routes routes.json --middlewares ./middleware.js",
-    "start:deploy": "yarn start:gen && yarn start:prod"
+    "gen:prod": "node generator.js",
+    "gen:dev": "cross-env NODE_ENV=development node generator.js",
+    "start": "yarn gen:prod && cross-env NODE_ENV=production PORT=8080 node .",
+    "start:dev": "yarn gen:dev && cross-env NODE_ENV=development PORT=8080 node ."
```

Comment:
perhaps one more script, with

```javascript
 "start:deploy": "yarn gen:prod && yarn start"
```

Reply:
Why tho? `yarn start` does the same thing

Reply:
My bad didn't see `node .` at the end